dotnet/samples/TelemetryExample/TelemetryExample.csproj

```diff
@@ -9,13 +9,14 @@
     <ImplicitUsings>disable</ImplicitUsings>
     <IsPackable>false</IsPackable>
     <!-- Suppress: "Declare types in namespaces", "Require ConfigureAwait" -->
-    <NoWarn>CA1050;CA1707;CA2007;VSTHRD111,SKEXP0050,SKEXP0054,SKEXP0060</NoWarn>
+    <NoWarn>CA1050;CA1707;CA2007;CS1591;VSTHRD111,SKEXP0050,SKEXP0054,SKEXP0060</NoWarn>
```

Comment:
Is there a reason why you don't want to get warnings for "Missing XML comment for publicly visible type or member" anymore?

Reply:
Because some helpers don't have XML comments, such as `TestConfiguration.cs`.